app/views/Export/ExportIntro.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+
+import getHealthcareAuthorities from '../../store/actions/healthcareAuthorities/getHealthcareAuthoritiesAction';
+import healthcareAuthorityOptionsSelector from '../../store/selectors/healthcareAuthorityOptionsSelector';
+import { useAssets } from '../../TracingStrategyAssets';
+import ExportTemplate from './ExportTemplate';
+import { Icons } from '../../assets';
+
+export const ExportIntro = ({ navigation }) => {
+  const { t } = useTranslation();
+  const { exportStartTitle, exportStartBody } = useAssets();
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getHealthcareAuthorities());
+  }, [dispatch]);
+  const authorities = useSelector(healthcareAuthorityOptionsSelector);
+  const selectedAuthorityDummy = authorities[0];
+
+  const onNext = () =>
+    navigation.navigate('ExportCodeInput', {
+      selectedAuthority: selectedAuthorityDummy,
+    });
+  const onClose = () => navigation.navigate('SettingsScreen');
```

Comment:
```suggestion
  const onClose = () => navigation.goBack();
```
minor thing, but if this is the first screen its probably better to pop a screen off than to do a forward navigation.

Reply:
Love it. Will make this change.